src/stores/models/WalletAddress.js

```diff
@@ -1,9 +1,11 @@
+import { observable } from 'mobx';
+
 import { satoshiToDecimal } from '../../helpers/utility';
 
 export default class WalletAddress {
-  address = ''
-  qtum = 0
-  bot = 0
+  @observable address = '';
```

Comment:
semicolon or not?

Reply:
it seems to be ok without the semi colon. force of habit to add it. I prefer it there. what about you?

Reply:
sounds good. ya i like semicolons too 👍